Documentation/Permissions.rst

```diff
@@ -6,6 +6,11 @@ Permissions
 
 * All commands must be performed on the MCVirt node, which can be accessed via SSH using LDAP credentials.
 
+* To run MCVirt commands as a superuser you must either:
+
+  * Have your username included in the superusers section in the configuration file
+  
+  * Be logged in as root.

```

Comment:
Would you be able to add a comment that MCVirt must always be run using SUDO and that MCVirt determines the permissions based on the username of the user that has run sudo. Maybe give an example of a sudo configuration that could be used, using NOPASSWD and limits the sudo configuration to just the MCVirt executable script?